packages/create-cloudflare/src/helpers/packageManagers.ts

```diff
@@ -25,7 +25,7 @@ export const detectPackageManager = () => {
 	if (process.env.TEST_PM && process.env.TEST_PM_VERSION) {
 		name = process.env.TEST_PM as PmName;
 		version = process.env.TEST_PM_VERSION;
-		process.env.npm_config_user_agent = name;
+		process.env.npm_config_user_agent = `${name}/${version}`;
```

Comment:
This was an attempt to get `yarn` working with `create-vite` but something appears to be stripping the version between us running `pnpx` and `create-vite` being run.

Reply:
should this change then be reverted?